packages/data-service/src/data-service.spec.ts

```diff
@@ -1200,18 +1213,13 @@ describe('DataService', function () {
         dataService.dropView(`${testDatabaseName}.myView`, done);
       });
 
-      it('returns 0 documents because the view has been dropped', function (done) {
-        dataService.count(
+      it('returns 0 documents because the view has been dropped', async function () {
+        const count = await dataService.count(
           `${testDatabaseName}.myView`,
           {},
-          {},
-          function (err, _count) {
-            if (err) return done(err);
-
-            assert.equal(_count, 0);
-            done();
-          }
+          {}
         );
+        assert.equal(count, 0);
```

Comment:
I realize this is existing code, but this should probably be

```suggestion
        expect(count).to.equal(0);
```

(and if not, it should probably be `assert.strictEqual` instead of `assert.equal`)